docs/src/content/upvote.js

```diff
@@ -1,13 +1,10 @@
 import ROUTES from '../route';
 
-const upvote = {
-  title: '👍 Upvote the features you want to get prioritized',
-  description:
-    'Want to request a feature? Head over to our repo and if not already listed, feel free to open an issue explaining the use case.',
-  action: {
-    href: ROUTES.toolpadUpvote,
-    label: 'Upvote',
-  },
-};
+export const title = '👍 Upvote the features you want to get prioritized';
```

Comment:
Did you change this pattern to enable tree shaking?
Eventually we could have other files follow the same pattern then for things to be consistent, but it's probably not a high priority.

Reply:
I did this to allow the `<Banner>` component to be used in the docs as well, using the `{{ component: ...}}` syntax, where I wasn't aware of a method to pass an object as props here https://github.com/mui/mui-toolpad/pull/1319#discussion_r1021933417

Reply:
Ah, sounds good then!